sending-emails-from-a-service/tests/test.bal

```diff
@@ -118,7 +118,7 @@ public client class MockHttpClient {
     isolated resource function post [http:PathParamType... path](http:RequestMessage message, 
             map<string|string[]>? headers = (), 
             string? mediaType = (), 
-            http:TargetType targetType = <>, *http:QueryParams params) 
+            http:TargetType targetType = http:Response, *http:QueryParams params) 
```

Comment:
Did this compile previously?

Reply:
Yes it did. After updating the version, got an error saying `cannot use an inferred typedesc default with a parameter on which the return type does not depend on`.